### PR TITLE
Make data list consistent with table 

### LINKED ISSUE #2185: Inconsistency between data lists and tables
Data lists and table have some differences that should probably be the same - eg: borders and border colors

_Differences illustrated in these slides and document:_ https://docs.google.com/presentation/d/1fLp8424_ykHEyS5K2kkGsfklE3tfIQCBy0eTu1G19go/edit?usp=sharing & https://docs.google.com/spreadsheets/d/1vqv6I00urK2YIyKZB320lUcYAhy54GVnfjXyqQgE2PA/edit?usp=sharing

**Symbol wide**
In multiples instance both in Desktop and Mobile view, border colors for data list rows are #D2D2D2, but #EDEDED for tables

**Expandable table**
1. Line between expandable row and content row is EDEDED in one, and D2D2D2 in the other
2. Data lists have a line at the bottom of the content row (both with padding and not), where as table content rows don’t have a border line at the bottom

### PR DESCRIPTION
**Pull request by christiemolloy**
fixes #2185 

When reviewing, refer to #2185 for the slides that @mmenestr made!

Screen Shot 2019-09-19 at 3 31 56 PM
Screen Shot 2019-09-19 at 3 32 38 PM
Screen Shot 2019-09-19 at 3 33 44 PM

### CODE REVIEW

**Comment by mcoker** [comment]
Sorry I'm probably just missing this somewhere, but is there a list of what needs to change to what? I'm still seeing differences in this PR, like this:

#### table
Screen Shot 2019-09-23 at 5 15 08 PM

#### data list
Screen Shot 2019-09-23 at 5 15 13 PM

**Comment by christiemolloy** [comment]
@mcoker here is the link to the slide share that Margo made: https://docs.google.com/presentation/d/1fLp8424_ykHEyS5K2kkGsfklE3tfIQCBy0eTu1G19go/edit#slide=id.g5e8e11457d_0_0 and I updated that bottom-border. Do you see anything else?

**Comment by mcoker** [comment]
@christiemolloy I don't see anywhere in the linked spreadsheet which values are correct - it's just pointing out differences. Same with https://github.com/patternfly/patternfly-next/issues/2185, it just lists the differences but doesn't say which is correct.

**Comment by christiemolloy** [comment]
Sorry, the values in the table are correct @mcoker

File: src/patternfly/components/DataList/data-list.scss [hunk 1]
```diff
@@ -113,7 +116,6 @@
   list-style-type: disc;
   background-color: var(--pf-c-data-list--BackgroundColor);
   border-top: var(--pf-c-data-list--BorderTopWidth) solid var(--pf-c-data-list--BorderTopColor);
-  border-bottom: var(--pf-c-data-list--BorderBottomWidth) solid var(--pf-c-data-list--BorderBottomColor);
```
Comment (mcoker) [inline src/patternfly/components/DataList/data-list.scss]: still need to add this line back in so it isn't a breaking change.

**Comment by mcoker** [comment]
A couple of questions that are outside the scope of this PR.

* Should we display the thick mobile borders between rows on data list when it breaks to a responsive view the same way we do with table?
* Should the same border between the expandable row and expanded content exist between data list and table?

datalist
Screen Shot 2019-09-24 at 11 09 54 AM

table

Screen Shot 2019-09-24 at 11 09 49 AM

**Comment by christiemolloy** [comment]
@mcoker great questions. The examples for data-list/table are here: https://sketch.cloud/s/R9k28/a/rWLoQP however they don't demonstrate every single example that we have so I think it would be great for me to sit down with a designer and go through all the examples that we have and make sure we're consistent across design and dev.

**Comment by christiemolloy** [comment]
@gdoyle1 added "update symbol library" label. I'm going to add 30 minutes on your calendar so that I can help you update the library!

**Comment by christiemolloy** [comment]
@mattnolting @mcoker fixed the ::before border

Screen Shot 2019-09-25 at 10 06 35 AM

**Comment by gdoyle1** [comment]
> @gdoyle1 added "update symbol library" label. I'm going to add 30 minutes on your calendar so that I can help you update the library!

Ah great thanks! Talk to you Friday about it

**Pull request merged**